src/webviews/mongoClusters/collectionView/components/toolbar/ToolbarMainView.tsx

```diff
@@ -38,6 +44,20 @@ const ToolbarQueryOperations = (): JSX.Element => {
             ...prev,
             currrentQueryDefinition: { ...prev.currrentQueryDefinition, queryText: queryContent, pageNumber: 1 },
         }));
+
+        trpcClient.common.reportEvent
+            .mutate({
+                eventName: 'executeQuery',
+                properties: {
+                    ui: 'button',
+                },
+                measurements: {
+                    queryLenth: queryContent.length,
+                },
+            })
+            .catch((error) => {
+                console.error('Failed to report query event:', error);
```

Comment:
If this is error then either it should show message to user or send to telemetry. If this is debug info, please use console.debug or log.